doc-content/enterprise-only/employee-rostering/optashift-ER-building-commandline-db-proc.adoc

```diff
@@ -17,7 +17,9 @@ If you use the command line to build the employee rostering starter application
 +
 [source,bash]
 ----
-java -jar optaweb-employee-rostering-standalone-*-exec.jar --spring.profiles.active=production
+java -jar quarkus-app/quarkus-run.jar --
+//How should this change for Quarkus?
```

Comment:
```
java -Dquarkus.datasource.username=<DATABASE_USER> -Dquarkus.datasource.password=<DATABASE_PASSWORD> -Dquarkus.datasource.jdbc.url=<DATABASE_URL> -jar optaweb-employee-rostering-standalone/target/quarkus-app/quarkus-run.jar
```

Important note: `quarkus.datasource.db-kind` is fixed at build-time and defaults to H2. If they want to use postgres, they need to rebuild the standalone module with `mvn clean install -DskipTests -Dquarkus.profile=postgres`